pydantic/types.py

```diff
@@ -1550,6 +1551,8 @@ class User(BaseModel):
     #> username='scolvin' password=SecretStr('**********')
     print(user.password.get_secret_value())
     #> password1
+    print(SecretStr('password'), SecretStr(''))
+    #> '**********' ''
```

Comment:
```suggestion
    print((SecretStr('password'), SecretStr('')))
    #> ('**********', '')
```

Reply:
It shows the repr when you make it a tuple, but anyway I still made this change